jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh

```diff
@@ -1,6 +1,7 @@
 # 55-swift-sproxyd.sh - Devstack extras script to configure s-object with swiftsproxyd driver
 
 function install_sproxyd_driver {
+    sudo pip install https://github.com/scality/scality-sproxyd-client/archive/master.tar.gz
```

Comment:
Can this address be retrieved from `requirements.txt` instead of duplicating it? That way, whenever we pin a certain version in `requirements.txt`, the tests will run against an installation with this version as well.